### PR TITLE
Bump activities

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +0,0 @@
-engine-strict=true
```
Comment (jeroach) [inline .npmrc]: @dlockhart this got deleted when I did the `rm -rf node_modules` and `npm i` 

Was that intended?

File: package-lock.json [hunk 2]
```diff
@@ -3576,7 +3576,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#8a013dd7c98d2f1a681850b693186619a9e135d9",
+      "version": "github:BrightspaceHypermediaComponents/activities#8ed5de1042d70965ded509bacb18c10b05a58d2d",
```
Comment (jeroach) [inline package-lock.json]: Activities:
https://github.com/BrightspaceHypermediaComponents/activities/commit/8ed5de1042d70965ded509bacb18c10b05a58d2d

File: .npmrc (hunk 1 above)
Comment (dlockhart) [reply .npmrc]: I don't think it should go away... AFAIK it's there to make sure NPM stays `<=6.8.0` to avoid all the issues we get with higher versions.

Comment (jeroach) [reply .npmrc]: kk, thanks - will put it back

File: package-lock.json (hunk 2 above)
Comment (AlexBedley) [reply package-lock.json]: Github even lets you see the difference between 2 commits: https://github.com/BrightspaceHypermediaComponents/activities/compare/8a013dd7c98d2f1a681850b693186619a9e135d9...8ed5de1042d70965ded509bacb18c10b05a58d2d

Comment (jeroach) [reply package-lock.json]: cool :)

**Pull request merged**